src/app/mail/mail-sidebar/compose-mail/compose-mail.component.ts

```diff
@@ -415,6 +427,10 @@ export class ComposeMailComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   sendEmail() {
+    if (!this.selectedMailbox.is_enabled) {
```

Comment:
@Waseem826 please don't show disabled email address in dropdown, so that user don't have the option to select it.